examples/js/loaders/GLTFLoader.js

```diff
@@ -913,6 +913,30 @@ THREE.GLTFLoader = ( function () {
 	GLTFCubicSplineInterpolant.prototype = Object.create( THREE.Interpolant.prototype );
 	GLTFCubicSplineInterpolant.prototype.constructor = GLTFCubicSplineInterpolant;
 
+	GLTFCubicSplineInterpolant.prototype.copySampleValue_ = function ( index ) {
+
+		// Copies a sample value to the result buffer. See description of glTF
+		// CUBICSPLINE values layout in interpolate_() function below.
+
+		var result = this.resultBuffer,
+			values = this.sampleValues,
+			valueSize = this.valueSize,
+			offset = index * valueSize * 3 + valueSize;
+
+		for ( var i = 0; i !== valueSize; i ++ ) {
+
+			result[ i ] = values[ offset + i ];
+
+		}
```

Comment:
Let me confirm my understanding. Is here copying spliteVertex_index in [ inTangent_1, splineVertex_1, outTangent_1, inTangent_2, splineVertex_2, outTangent_2, ... ] into `result`?

Reply:
That's right. 👍

If we don't override this method, it does:

```js
var result = this.resultBuffer,
  values = this.sampleValues,
  stride = this.valueSize,
  offset = index * stride;

for ( var i = 0; i !== stride; ++ i ) {

  result[ i ] = values[ offset + i ];

}

return result;
```

... which as you might imagine, does not work properly since our interpolant's valueSize and stride are not the same. 😅

Reply:
OK OK

I don't understand `beforeStart_()` and `afterEnd_()` in `Interpolant` yet, so wondering if it's ok to ignore the second and third arguments `t0` and `t1`.

https://github.com/mrdoob/three.js/blob/dev/src/math/Interpolant.js#L117

Reply:
Based on https://github.com/mrdoob/three.js/blob/dev/src/math/Interpolant.js#L248-L256, I think it is consistent with current behavior. But that is odd..

Reply:
Hm, so merging this PR so far and looking into `beforeStart_()` and `afterEnd_()` later? At least, this PR seems to follow the current `Interpolant` behavior...